<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="AssignmentDemo" duration-ms="29075" started-at="2021-02-04T12:39:28 IST" finished-at="2021-02-04T12:39:57 IST">
    <groups>
    </groups>
    <test name="CreateNewAccount" duration-ms="29075" started-at="2021-02-04T12:39:28 IST" finished-at="2021-02-04T12:39:57 IST">
      <class name="com.demo.tests.CreateNewAccountTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.demo.tests.CreateNewAccountTest@5025a98f]" name="setUp" is-config="true" duration-ms="2645" started-at="2021-02-04T12:39:28 IST" finished-at="2021-02-04T12:39:31 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="createNewAccountTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.demo.tests.CreateNewAccountTest@5025a98f]" name="createNewAccountTest" duration-ms="22181" started-at="2021-02-04T12:39:31 IST" data-provider="CreateNewAccount" finished-at="2021-02-04T12:39:53 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[Test_FirstName]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Test_LastName]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[1234554321]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Test@123]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[15,Jun,1984]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[Male]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The error is not matching expected [Too many users have this phone number listed as pending.] but found [Too many users have this phone number listed as pending.3344]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The error is not matching expected [Too many users have this phone number listed as pending.] but found [Too many users have this phone number listed as pending.3344]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at com.demo.tests.CreateNewAccountTest.createNewAccountTest(CreateNewAccountTest.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createNewAccountTest -->
        <test-method status="PASS" signature="quiteBrowser()[pri:0, instance:com.demo.tests.CreateNewAccountTest@5025a98f]" name="quiteBrowser" is-config="true" duration-ms="4064" started-at="2021-02-04T12:39:53 IST" finished-at="2021-02-04T12:39:57 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quiteBrowser -->
      </class> <!-- com.demo.tests.CreateNewAccountTest -->
    </test> <!-- CreateNewAccount -->
  </suite> <!-- AssignmentDemo -->
</testng-results>
